### PR TITLE
tests: additional smart contracts deploy and update

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: pytest_tests/testsuites/contract/test_contract/deploy/neo-go.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: testctr
+sourceurl: http://example.com/
+safemethods: []
+supportedstandards: []
+events:
+    - name: Hello world!
```
Comment (roman-khimov) [inline pytest_tests/testsuites/contract/test_contract/deploy/neo-go.yml:6]: Can be removed, there are no events in the code.

File: pytest_tests/testsuites/contract/test_contract/update/neo-go.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: testctr
+sourceurl: http://example.com/
+safemethods: []
+supportedstandards: []
+events:
+    - name: Hello world!
```
Comment (roman-khimov) [inline pytest_tests/testsuites/contract/test_contract/update/neo-go.yml:6]: Also missing.

File: pytest_tests/testsuites/contract/test_contract/deploy/neo-go.yml (hunk 1 above)
Comment (evgeniiz321) [reply pytest_tests/testsuites/contract/test_contract/deploy/neo-go.yml]: fixed

File: pytest_tests/testsuites/contract/test_contract/update/neo-go.yml (hunk 2 above)
Comment (evgeniiz321) [reply pytest_tests/testsuites/contract/test_contract/update/neo-go.yml]: fixed

**Pull request merged**